.packit.yaml

```diff
@@ -103,3 +103,25 @@ jobs:
       - fedora-latest # branched version, rawhide updates are created automatically
       - fedora-stable
       - epel-9
+
+  # vm image build
+  - job: copr_build
+    trigger: pull_request
+    targets:
+      - fedora-all
+    project: vm-image-ogr-demo
```

Comment:
I am worried this will collide with the statuses from the other `copr_build`. (If it is just for the demo, you can delete the original one.)

Reply:
I try to use the `identifier` key